components/Header.tsx

```diff
@@ -52,7 +52,7 @@ const Header = () => {
                         Positions
                         </div>
                     </Link>
-                    <Link href={'/pools'}>
+                    <Link href={'/pools/pools'}>
```

Comment:
Why is this link `pools/pools`? It should be just `/pools` because it's the main page of pools

Reply:
We have another /pools/createpool page that Daniel wanted for us to have to create pools via the CFMM address.

Reply:
maybe changing the route to `pools/menu` to avoid redundancy will work.

Reply:
@UsamaSaleem14 can you make this change?